config/default/kustomization.yaml

```diff
@@ -22,6 +22,7 @@ resources:
 - ../crds/cluster_v1alpha1_machine.yaml
 - ../crds/cluster_v1alpha1_machinedeployment.yaml
 - ../crds/cluster_v1alpha1_machineset.yaml
+- ../crds/cluster_v1alpha1_machineclass.yaml
```

Comment:
These look alphabetically sorted, in which case this should go above machinedeployment.yaml.